python/pyproject.toml

```diff
@@ -135,7 +136,7 @@ all = ["transformers", "sentence-transformers", "qdrant-client", "chromadb", "py
 
 azure = ["azure-ai-inference", "azure-search-documents", "azure-core", "azure-identity", "azure-cosmos", "msgraph-sdk"]
 chromadb = ["chromadb"]
-hugging_face = ["transformers", "sentence-transformers"]
+hugging_face = ["datasets", "transformers", "sentence-transformers"]
```

Comment:
Is this only for this sample? Of so I think just adding the pip install comment in the sample itself is enough, or do you see a benefit of having it available more broadly?

Reply:
Removed, and added the requirement to the README